### PR TITLE
[Keyboard] Add archerkeyboard/desire65

### PR DESCRIPTION
**Pull request by akyp**
## Description

Add new keyboard archerkeyboard/desire65

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/archerkeyboard/desire65/config.h
Comment (waffle87) [inline keyboards/archerkeyboard/desire65/config.h:1]: This file should be removed.

File: keyboards/archerkeyboard/desire65/readme.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+![desire65](https://i.imgur.com/bTjvGzC.jpg)
```
Comment (waffle87) [inline keyboards/archerkeyboard/desire65/readme.md:1]:
```suggestion
# desire65

![desire65](https://i.imgur.com/bTjvGzC.jpg)
```

File: keyboards/archerkeyboard/desire65/rules.mk
Comment (waffle87) [inline keyboards/archerkeyboard/desire65/rules.mk:1]: Given that the breaking change cycle is occurring in the near future, this file can be removed, and the `info.json` renamed to `keyboard.json`.

**Review by drashna — APPROVED** [review]

File: keyboards/archerkeyboard/desire65/keyboard.json [hunk 2]
```diff
@@ -0,0 +1,197 @@
+{
+    "manufacturer": "archerkeyboard",
+    "keyboard_name": "desire65",
+    "maintainer": "akyp",
+    "backlight": {
+        "driver": "software",
+        "levels": 8,
+        "pin": "GP18"
+    },
```
Comment (zvecr) [inline keyboards/archerkeyboard/desire65/keyboard.json:9]:
```suggestion
    "indicators": {
        "caps_lock": "GP18"
    },
```

File: keyboards/archerkeyboard/desire65/keyboard.json [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "backlight": true,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
```
Comment (zvecr) [inline keyboards/archerkeyboard/desire65/keyboard.json:16]:
```suggestion
        "bootmagic": true,
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/archerkeyboard/desire65/keyboard.json (hunk 3 above)
Comment (akyp) [reply keyboards/archerkeyboard/desire65/keyboard.json]: Changed as requested

File: keyboards/archerkeyboard/desire65/keyboard.json (hunk 2 above)
Comment (akyp) [reply keyboards/archerkeyboard/desire65/keyboard.json]: Changed as requested

**Review by drashna — APPROVED** [review]

**Review by tzarc — APPROVED** [review]

**Pull request merged**